windows/src/engine/tsysinfo/Makefile

```diff
@@ -32,8 +32,8 @@ backup:
     $(WZZIP) $(BUILD)\engine\tsysinfo.zip  $(BACKUPDEFAULTS) tsysinfo.exe
 
 install:
-    $(COPY) $(PROGRAM)\engine\tsysinfo.exe "C:\Program Files (x86)\Common Files\Tavultesoft\Keyman Engine 9.0"
-    $(COPY) $(ROOT)\src\engine\tsysinfo\*.xsl "C:\Program Files (x86)\Common Files\Tavultesoft\Keyman Engine 9.0"
+    $(COPY) $(PROGRAM)\engine\tsysinfo.exe "C:\Program Files (x86)\Common Files\Keyman\Keyman Engine 10.0"
+    $(COPY) $(ROOT)\src\engine\tsysinfo\*.xsl "C:\Program Files (x86)\Common Files\Keyman\Keyman Engine 10.0"
```

Comment:
Given we are fixing all these now, let's define variables in Defines.mak for the three install paths, e.g. `INSTALLPATH_KEYMANDESKTOP`, `INSTALLPATH_KEYMANDEVELOPER`, `INSTALLPATH_KEYMANENGINE`. Then we can avoid changing all of these makefiles again in the future.

You may be able to make it slightly more robust as well by using the appropriate environment variables instead of `C:\Program Files (x86)\Common Files` and `C:\Program Files (x86)` . Research and test for these...